src/actions/starMessage.js

```diff
@@ -0,0 +1,26 @@
+import { getRoomsChannel } from '../reducers/rooms'
+
+export const starMessage = (messageId) => ({
+  type: 'STAR_MESSAGE',
+  messageId: messageId
+})
+
+export const unstarMessage = (messageId) => ({
+  type: 'UNSTAR_MESSAGE',
+  messageId: messageId
+})
+
+export const replaceStarMessages = (data) => ({
+  type: "REPLACE_STAR_MESSAGES",
+  starredMessages: data
+})
+
+export const submitStar = (messageId) => (dispatch, getState) => {
+  const rooms = getRoomsChannel(getState())
+  return rooms.push('starred_message:create', messageId)
+}
+
+export const deleteStar = (messageId) => (dispatch, getState) => {
+  const rooms = getRoomsChannel(getState())
+  return rooms.push('starred_message:delete', messageId)
+}
```

Comment:
Both `deleteStar` and `submitStar` are asynchronous actions. To enable actions to be taken after the server responds, let's update both of these to take optional callback methods.

For an example of how to implement this, see [userSubscriptions](https://github.com/oberon-chat/chat-client/blob/master/src/actions/userSubscriptions.js#L6).